src/Sulu/Bundle/PageBundle/Resources/js/index.js

```diff
@@ -44,7 +44,7 @@ initializer.addUpdateConfigHook('sulu_page', (config: Object, initialized: boole
     );
 
     if (config.versioning) {
-        fieldRegistry.add('page_settings_versions', PageSettingsVersions);
+        fieldRegistry.add('settings_versions', SettingsVersions);
```

Comment:
not sure if we should move this to another bundle now that it is reusable 🤔

Reply:
Hm, not sure about this... I think I would leave it here for the time being.

The thing is that the `versioning` parameter checked in the line before is also in the PageBundle, and we have to avoid registering it when this configuration is set to false, in order for the `on-invalid` attribute in the XML to work correctly.

I am also not sure where to move it instead, and I think the ArticleBundle is already depending a lot on the PageBundle (am I right?). Don't like that fact, but I think it is still better to keep it this way, than introducing another dependency right away.

Reply:
is alright for me 🙂